.coderabbit.yaml

```diff
@@ -0,0 +1,30 @@
+language: "en"
+early_access: false
+reviews:
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: true
+  review_status: true
+  collapse_walkthrough: false
+  path_filters:
+    - "!**/.xml"
+    - "great_expectations/**"
+  # path_instructions:
+  #     - path: "**/*.js"
+  #       instructions: "Review the JavaScript code for conformity with the Google JavaScript style guide, highlighting any deviations."
+  #     - path: "tests/**/*"
+  #       instructions: |
+  #           "Assess the unit test code employing the Mocha testing framework. Confirm that:
+  #           - The tests adhere to Mocha's established best practices.
+  #           - Test descriptions are sufficiently detailed to clarify the purpose of each test."
```

Comment:
Do we need this information? Might also be nice to have a quick summary at the top stating what we're expecting this config to do (summary only)